app/harvesters/nara_harvester.rb

```diff
@@ -0,0 +1,126 @@
+##
+# A harvester for NARA's API
+#
+# @see Krikri::Harvesters::ApiHarvester
+class NaraHarvester < Krikri::Harvesters::ApiHarvester
+  DEFAULT_URI = 'https://catalog.archives.gov/api/v1'
+  DEFAULT_NAME = 'nara'
+  DEFAULT_BATCHSIZE = 10
+  DEFAULT_ID_FILENAME = '/var/tmp/nara_ids'
+  DEFAULT_PARAMS = {
+    'pretty' => 'false',
+    'resultTypes' => 'item,fileUnit',
+    'objects.object.@objectSortNum' => '1'
+  }
+
+  ##
+  # Initialize, and set default options as appropriate.
+  #
+  # @param opts [Hash] a hash of options as defined by {.expected_opts}
+  #
+  # @example
+  #    Typical instantiation, good for most cases:
+  #        NaraHarvester.new
+  #    Specifying custom parameters:
+  #        NaraHarvester.new(api: {'some_query_param' => 'abc'}, batchsize: 15)
+  #
+  # Parameters to 'opts':
+  # - uri:        See Krikri::Harvester#initialize.
+  #               Defaults to "https://catalog.archives.gov/api/v1"
+  # - name:       See Krikri::Harvester#initialize.  Defaults to "nara"
+  # - batchsize:  The number of records to fetch with each API request.
+  #               Defaults to 10.
+  # - id_fname:   The file name of the Heidrun::IDSource that drives the
+  #               harvest and governs which records to fetch.  Defaults to
+  #               /var/tmp/nara_ids.
+  # - id_fh:      A filehandle to use with Heidrun::IDSource, mostly useful for
+  #               automated testing or console usage.  Optional.
+  #
+  # For other parameters, see Krikri::Harvester#initialize.  
+  #
+  # Parameters for API requests can be specified with the :api key, but have
+  # default values and this should not usually be necessary. See
+  # https://github.com/usnationalarchives/Catalog-API/blob/master/search_and_export.md
+  #
+  # @raise Error::ENOENT  If the IDSource file is missing
```

Comment:
I'm thinking this type spec won't parse, per http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md#Declaring_Types

`# @raise [Error::ENOENT] If the IDSource file is missing`